src/Core/src/Handlers/Label/LabelHandler.Windows.cs

```diff
@@ -53,8 +53,15 @@ public static void MapText(ILabelHandler handler, ILabel label) =>
 		public static void MapTextColor(ILabelHandler handler, ILabel label) =>
 			handler.PlatformView?.UpdateTextColor(label);
 
-		public static void MapCharacterSpacing(ILabelHandler handler, ILabel label) =>
+		public static void MapCharacterSpacing(ILabelHandler handler, ILabel label)
+		{
+			if (handler.IsConnectingHandler() && label.CharacterSpacing == 0)
```

Comment:
LGTM, avoids setting the default CharacterSpacing value during the initial setup of the control or when is not necessary, which should give a performance boost in scenarios with many labels.

Running a build to verify if not have any impact in some test scenario related with Label.CharacterSpacing.